File: peps/src/peps/equity.java
package peps;

public class equity {
	public String name;
	public double spot;
	public double vol;
	public double[] historiq;
	

	public equity(String namep, double spotp, double volp ) {
		name = namep;
		spot = spotp;
		vol = volp;
		historiq = new double[4];
	}
	
	public String getName() {
	    return this.name;
	}
	
	public double getspot() {
	    return this.spot;
	}

	public double getvol() {
	    return this.vol;
	}
}

File: peps/Peps/Equity.cpp
#include "StdAfx.h"
#include "Equity.h"


Equity::Equity(double spot ,double vol, std::string Ename)
{
	value = spot;
	volatility = vol;
	name = Ename;
}
Equity::Equity(double spot ,double vol, int Ename)
{
	value = spot;
	volatility = vol;
}


Equity::~Equity(void)
{
}
